src/commands/createStaticWebApp/createStaticWebApp.ts

```diff
@@ -22,11 +23,24 @@ export async function createStaticWebApp(context: IActionContext, node?: Subscri
     ext.outputChannel.appendLog(createdSs);
 
     const showActionsMsg: MessageItem = { title: 'Show Actions' };
+    const cloneRepoMsg: MessageItem = { title: localize('cloneRepo', 'Clone Repo') };
+    const msgItems: MessageItem[] = [showActionsMsg];
+
+    if (context.telemetry.properties.gotRemote === 'false') {
+        // only ask to clone if we didn't detect the remote for creation
+        msgItems.unshift(cloneRepoMsg);
+    }
+
     // don't wait
-    window.showInformationMessage(createdSs, showActionsMsg).then(async (result) => {
-        if (result === showActionsMsg) {
-            await showActions(context, ssNode);
-        }
+    window.showInformationMessage(createdSs, ...msgItems).then(async (result) => {
+        await callWithTelemetryAndErrorHandling('postCreateStaticWebApp', async (context2: IActionContext) => {
```

Comment:
I don't quite understand the scenario for "Clone Repo" here. This means users have never tried their code locally before deploying to Azure? That just seems uncommon to me. Even if I created a new GitHub repo in this flow, it seems more likely I need to do a `git remote set-url`, not a `git clone`

Reply:
Well, if you look at the issue this fixes, that user was expecting to have the project in his local workspace.  He made a new repo through our extension, thus he needed to clone it.

That being said, there have been many cases where I linked a repo that I had hosted on GitHub, but not on my local machine.  This is also a way to guide users to the next step, which is having the project opened in their local workspace and commit changes.

Reply:
Asked the user for more info in the issue. Are there any hints here from our telemetry? For example: how often people use "Clone" from the right click menu, the percent of people that create a new repo, the percent that already had a remote

> That being said, there have been many cases where I linked a repo that I had hosted on GitHub, but not on my local machine. 

Well is that because you were forking a bunch of sample apps for testing instead of creating your own?

> This is also a way to guide users to the next step, which is having the project opened in their local workspace and commit changes.

The debate is what's the "next step". I would argue many users already have a project open before they try to deploy. So the next step would either be just commit/push

Reply:
> Are there any hints here from our telemetry? For example: how often people use "Clone" from the right click menu, the percent of people that create a new repo, the percent that already had a remote

Well, clone is a new feature that wouldn't have telemetry yet because it's gonna be included in this next release.  We can check how often the remote is detected though.

> The debate is what's the "next step". I would argue many users already have a project open before they try to deploy. So the next step would either be just commit/push

Well, the clone button only appears _if_ we didn't detect a remote for the user.  So this could be a non-issue.  But I do think if we can detect that they don't have their repo opened, then we should give them the option to bring it down.

That being said, I think a logical feature to add to this is to have the ability of creating a new repo _with_ the local project that they have opened.